Update the way copilot edit is started (#101)
github-samples/pets-workshop: content/full-day/6-code.md

   - **server/app.py**
   - **server/test_app.py**
   - **client/src/components/DogList.svelte** 
2. Open GitHub Copilot Chat.
3. Switch to edit mode by selecting **Edit** in the chat mode dropdown at the bottom of Chat view (should be currently **Ask**)

> 4. Ensure **Claude 3.7 Sonnet** is selected for the model.

Reviewer: ```suggestion
4. If available, select **Claude 3.7 Sonnet** for the model.
```

Model list can vary

5. Select **Add Context...** in the chat window.

> 6. Select **server/app.py**, **client/src/components/DogList.svelte** and **server/test_app.py** files (you need to select **Add context** for each file) 

Reviewer: Curious the rationale behind this update. I personally find it easier to go with the "add all open editors" option

Author: I would too, but it doesn't exist anymore. :) 

Unless I'm missing something very obvious

Reviewer: Weird. I just checked, and it's on the stable but not insiders. Let's roll with what you've got here

>   - **Hint**: if you type the file names, they will show up in the filter. You can also drag the files or right click file in explorer and select `Copilot -> Add File to Chat`)

Reviewer: Rather than separate bullet point let's have this as a note if we're going to include drag & drop. I'd also highlight this doesn't need to be all-inclusive of every option available

Author: Do you prefer the note to be "inside" the bullet point 6), or after all bullet points?

Reviewer: Let's go with "inside" bullet point 6

Reviewer: It will also make @peckjon happy as he wanted a callout to this previously. I personally never use the feature, but since there's now a second request apparently I'm the oddball

Author: Updated in 474fddb7d9810c33beeb75db99e448a214d44d0e
